src/WebApp/App_Data/jobs/triggered/EnvironmentSetup/init.py

```diff
@@ -1,11 +1,19 @@
-import os
 import urllib
 import zipfile
 from datetime import datetime
 from azure.storage.table import TableService, Entity, TablePermissions
 from azure.storage.blob import BlockBlobService
 from azure.storage.blob import PublicAccess
 from azure.storage.file import FileService
+import sys, os, time, glob
+import requests
+import json
+import uuid
+import json
+import random
+import markdown
+import jwt
```

Comment:
you certainly don't need markdown and jwt here. possibly other libraries as well.